Solar iOSTests/CorrectResults.swift

```diff
@@ -0,0 +1,1432 @@
+//
+//  Created on 07/06/2020.
+//  
+//
+
+import Foundation
+
+let correctResults = """
+[{
```

Comment:
I assume CorrectResults.json was replaced by CorrectResults.swift to avoid bundling a resource file? Xcode 12 and SPM 5.3 now supports bundle resources, so you can switch back to the JSON file to simplify the PR.

Reply:
Yes. the change was made to make it compatible with spm 5.2 that is with prior Xcode 12. Taking in consideration all the problems with Xcode 12 silicone architecture/carthage problems it wold make sense to make it compatible for some developers.

Reply:
If a downstream project is relying on SPM support, I don’t think it would be affected by Carthage’s lack of XCFramework support.

It would be possible to require Swift 5.3 and Xcode 12 if using SPM but maintain compatibility with Xcode 11 for now when using other package managers. For example, [MapboxDirections](https://github.com/mapbox/mapbox-directions-swift#system-requirements) says it requires “ Xcode 11 or above (Xcode 12 or above if using Swift Package Manager)”. I think this is a more maintainable solution going forward, because it’s only a matter of time before the user community moves on from Xcode 11.

Reply:
Ok. I have just update the pr to support spm 5.3 resources.

Reply:
Thanks!